Monika After Story/game/script-topics.rpy

```diff
@@ -8194,6 +8194,87 @@ label monika_whydoyouloveme:
 #    m 1k "That's why I love you, [player]!"
 #    return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_bored",
+            category=['misc'],
+            random=True,
+            prompt="Boredom"
+        )
+    )
+
+label monika_bored:
+    python:
+        unlockedgames = [
+            game
+            for game in persistent.game_unlocks
+            if persistent.game_unlocks[game]
+        ]
+
+        gamepicked = renpy.random.choice(unlockedgames)
+        display_picked = gamepicked
+
+        if gamepicked == "hangman" and persistent._mas_sensitive_mode:
+            display_picked = "word guesser"
+
+    if mas_isMoniAff(higher=True):
```

Comment:
Wrong order here. the `mas_isMoniLove()` condition will never be used because `mas_isMoniAff(higher=True)` is `True` at love.